python/pyspark/ml/evaluation.py

```diff
@@ -467,6 +470,11 @@ def setParams(
         kwargs = self._input_kwargs
         return self._set(**kwargs)
 
+    def isLargerBetter(self) -> bool:
+        if self.getMetricName() in ["r2", "var"]:
+            return True
+        return False
```

Comment:
```suggestion
        return self.getMetricName() in ["r2", "var"]
```

we can simplify it a bit